src/Uno.UI/UI/Xaml/Media/Acrylic/AndroidMaterialFrameRenderer.cs

```diff
@@ -0,0 +1,239 @@
+
+			_acrylicLayer?.Dispose();
+			_acrylicLayer = null;
+		}
+
+		private void UpdateCornerRadius()
+		{
+			_acrylicLayer?.SetCornerRadius(Context.ToPixels(MaterialFrame.CornerRadius));
+			_realtimeBlurView?.SetCornerRadius(Context.ToPixels(MaterialFrame.CornerRadius));
+		}
+
+		private void UpdateElevation()
+		{
+			if (MaterialFrame.MaterialTheme == MaterialFrame.Theme.Dark || MaterialFrame.MaterialTheme == MaterialFrame.Theme.AcrylicBlur)
+			{
+				ViewCompat.SetElevation(this, 0);
+				return;
+			}
+
+			bool isAcrylicTheme = MaterialFrame.MaterialTheme == MaterialFrame.Theme.Acrylic;
+
+			// we need to reset the StateListAnimator to override the setting of Elevation on touch down and release.
+			StateListAnimator = new Android.Animation.StateListAnimator();
+
+			// set the elevation manually
+			ViewCompat.SetElevation(this, isAcrylicTheme ? MaterialFrame.AcrylicElevation : MaterialFrame.Elevation);
+		}
+
+		private void UpdateLightThemeBackgroundColor()
+		{
+			if (MaterialFrame.MaterialTheme == MaterialFrame.Theme.Dark || MaterialFrame.MaterialTheme == MaterialFrame.Theme.AcrylicBlur)
+			{
+				return;
+			}
+
+			_mainDrawable.SetColor(MaterialFrame.LightThemeBackgroundColor.ToAndroid());
+		}
+
+		private void UpdateAcrylicGlowColor()
+		{
+			_acrylicLayer?.SetColor(MaterialFrame.AcrylicGlowColor.ToAndroid());
+		}
+
+		private void UpdateMaterialTheme()
+		{
+			switch (MaterialFrame.MaterialTheme)
+			{
+				case MaterialFrame.Theme.Acrylic:
+					SetAcrylicTheme();
+					break;
+
+				case MaterialFrame.Theme.Dark:
+					SetDarkTheme();
+					break;
+
+				case MaterialFrame.Theme.Light:
+					SetLightTheme();
+					break;
+
+				case MaterialFrame.Theme.AcrylicBlur:
+					SetAcrylicBlurTheme();
+					break;
+			}
+		}
+
+		private void SetAcrylicBlurTheme()
+		{
+			_mainDrawable.SetColor(Color.Transparent.ToAndroid());
+
+			this.SetBackground(_mainDrawable);
+
+			UpdateElevation();
+
+			EnableBlur();
+		}
+
+		private void SetDarkTheme()
+		{
+			DisableBlur();
+
+			_mainDrawable.SetColor(MaterialFrame.ElevationToColor().ToAndroid());
+
+			this.SetBackground(_mainDrawable);
+
+			UpdateElevation();
+		}
+
+		private void SetLightTheme()
+		{
+			DisableBlur();
+
+			_mainDrawable.SetColor(MaterialFrame.LightThemeBackgroundColor.ToAndroid());
+
+			this.SetBackground(_mainDrawable);
+
+			UpdateElevation();
+		}
+
+		private void SetAcrylicTheme()
+		{
+			if (_acrylicLayer == null)
+			{
+				_acrylicLayer = new GradientDrawable();
+				_acrylicLayer.SetShape(ShapeType.Rectangle);
+			}
+
+			UpdateAcrylicGlowColor();
+			UpdateCornerRadius();
+
+			_mainDrawable.SetColor(MaterialFrame.LightThemeBackgroundColor.ToAndroid());
+
+			LayerDrawable layer = new LayerDrawable(new Drawable[] { _acrylicLayer, _mainDrawable });
+			if (Android.OS.Build.VERSION.SdkInt >= Android.OS.BuildVersionCodes.M)
+			{
+				layer.SetLayerInsetTop(1, (int)Context.ToPixels(2));
+			}
+			else
+			{
+				System.Console.WriteLine(
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932087)